packages/golden_toolkit/README.md

```diff
@@ -109,6 +187,53 @@ await multiScreenGolden(
       );
 ```
 
+### multiDeviceGolden
+
+The multiDeviceGolden assertion is used to capture a golden of a single widget that has different simulated device sizes & characteristics inside. All stitched together in one single png file. As opposed to multiple in multiScreenGolden
+
+The typical use case is to validate that a UI is responsive for both phone & tablet in one single golden file.
+
+```dart
+testGoldens('multiDeviceGolden', (tester) async {
+    await multiDeviceGolden(
```

Comment:
Hi @coreysprague - thank you for taking the time to review this PR. Sorry i've not had a chance to reply. Been a hell of a week.

To be fair, this method was more of a change i did to help test this whole new api at work. Rather than me having to refactor all of our `multiScreenGolden` calls to use the new DeviceBuilder structure, i thought it'd be easier to do this function wrapper and a more simple find and replace.

As mentioned i the README, the same functionality definitely can be done by just using the DeviceBuilder with one scenario.

I'm happy to drop this if you'd like. Can always have it as an extension on the project side rather than package.

Reply:
Thanks @moonytoes29. Let's leave it as a project extension. I really do like the simplicity, but I think it might cause some confusion until we have a chance to consolidate and cleanup the APIs.

Reply:
removed.